packages/apps/shopify-api/docs/reference/webhooks/addHandlers.md

```diff
@@ -64,11 +64,6 @@ Fields to be included in the callback, defaulting to including all of them.
 
 Namespaces to be included in the callback, defaulting to including all of them.
 
-### subTopic
```

Comment:
We still have that field in the code, so I think we should keep the documentation around. We need to deprecate the feature as a whole, but maybe we can just mark it as deprecated here?